src/SFML/Window/DRM/DRMContext.cpp

```diff
@@ -204,8 +204,7 @@ m_config     (NULL),
 m_currentBO  (NULL),
 m_nextBO     (NULL),
 m_gbmSurface (NULL),
-m_width      (0),
-m_height     (0),
+m_size       (0, 0),
```

Comment:
Do we care about explicitly initializing vectors to zero when the default constructor already does this?